features/user/managing_customers/seeing_orders_statistics_on_customer_details_page.feature

```diff
@@ -0,0 +1,44 @@
+@customer_statistics
+Feature: Seeing customer's orders' statistics
+    In order to know how many orders a customer has placed, and what's their worth
+    As an Administrator
+    I want to be able to see customer's per channel orders' statistics on their details page
```

Comment:
`I want to be able to see orders statistics per channel for customers on their details page` wdyt?